crud/src/main/java/com/redhat/lightblue/rest/crud/cmd/RunSavedSearchCommand.java

```diff
@@ -70,15 +66,19 @@ public RunSavedSearchCommand(String searchName,
         this.from=from;
         this.to=to;
         this.params=properties;
+        this.metrics = metrics;
     }
 
     @Override
     public CallStatus run() {
+        RequestMetrics.Context savedSearchMetricCtx = metrics.startEntityRequest("savedsearch", entity, version);
+        RequestMetrics.Context findMetricCtx = null;
```

Comment:
I'm not sure we need to track both saved search and find for a saved search. The saved search metric count as a find, so you could argue that we can combine this with graphite queries if we want. Then, we don't have to worry about tracking two things in the code. This also technically allows us to do things like, compare non-saved searches with saved searches. If we put saved searches in finds, too, then it is sort of lossy.

Reply:
Removed redundant find monitoring.